pkg/core/native/policy.go

```diff
@@ -167,42 +188,47 @@ func (p *Policy) Metadata() *interop.ContractMD {
 
 // Initialize initializes Policy native contract and implements the Contract interface.
 func (p *Policy) Initialize(ic *interop.Context, hf *config.Hardfork, newMD *interop.HFSpecificContractMD) error {
-	if hf != p.ActiveIn() {
-		return nil
+	if hf == p.ActiveIn() {
```

Comment:
Does it work correctly in all cases (all HF active at 0, Domovoi at 0 with Echidna at X, D&E at X)?

Reply:
Yes, a couple of tests are added.